update readme (#104)
resource-watch/api-infrastructure: README.md

**Important**: this repo uses [git lfs](https://git-lfs.github.com/).

For a description of the setup, see the infrastructure [section](https://resource-watch.github.io/doc-api/developer.html#infrastructure-configuration) of the developer documentation.

## Setting up the AWS resources

To setup the cluster cloud resources, use the following command:

```shell script
cd ./terraform

> export CLOUDFLARE_API_KEY=<cloudflare api key> CLOUDFLARE_EMAIL=<cloudflare api key>

Reviewer: ```suggestion
export CLOUDFLARE_API_KEY=<cloudflare api key> CLOUDFLARE_EMAIL=<cloudflare account email address>
```

From here, there are multiple ways to proceed.

### SSH tunnel

Perhaps the most practical way to connect to the cluster is by creating an SSH tunnel that connects a local port to the cluster's API port, through the bastion. For this to work, a few things are needed:

- Copy the `kubectl_config` settings from above into your local `~/.kube/config`
- Modify the `server: https://<random string>.gr7.us-east-1.eks.amazonaws.com` line by adding `:4433` at the end, so it looks like this: `server: https://<random string>.gr7.us-east-1.eks.amazonaws.com:4433` (you can pick a different port if you want)

> - Modify your local `/etc/hosts` to include the following line: `127.0.0.1 <random string>.gr7.us-east-1.eks.amazonaws.com`

Reviewer: ```suggestion
- Modify your local `/etc/hosts` to include the following line: `127.0.0.1 <eks api endpoint url>`
```

```shell script

> ssh -N -L 4433:<random string>.gr7.us-east-1.eks.amazonaws.com:443 ubuntu@<bastion_hostname>

Reviewer: ```suggestion
ssh -N -L 4433:<eks api endpoint url>:443 <bastion user>@<bastion hostname>
```
